src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -127801,6 +127801,4340 @@ static T F2<T>()
                 Diagnostic(ErrorCode.WRN_NullReferenceReturn, "t2").WithLocation(13, 16));
         }
 
+        [Fact]
+        public void UnconstrainedTypeParameter_01()
```

Comment:
Is there a version of this test that verifies the behavior works in C# 9?

Reply:
Yes, this one test verifies the behavior of both C#8 and preview.

---
In reply to: [457839142](https://github.com/dotnet/roslyn/pull/45993#discussion_r457839142) [](ancestors = 457839142)